cmd/odf/devpreview/devpreview.go

```diff
@@ -0,0 +1,92 @@
+package devpreview
+
+import "github.com/spf13/cobra"
+
+const (
+	// Suffix for appending to short command description.
+	Suffix = " (developer preview)"
+
+	// Note for using as long command description.
+	Note = `
++------------------------------------------------------------------------------+
+|                                                                              |
+|    This command is a developer preview, unsupported and not fully tested.    |
+|    Please see the following document for more info on developer preview:     |
+|    https://access.redhat.com/support/offerings/devpreview.                   |
+|                                                                              |
++------------------------------------------------------------------------------+
+`
+	// AllowFlag is required to run developer preview comamands.
+	AllowFlag = "allow-developer-preview"
```

Comment:
Flag is bit descriptive, maybe just `--dev-preview` or `--allow-dev-preview`?

Reply:
Keeping as is to get more input from others.

Reply:
I thought we would use --allow-unsupported-commands? Dev Preview isn't necessarily obvious to customers

Reply:
The flag was removed since we hide the dr command now.